fenix/app/src/main/java/org/mozilla/fenix/onboarding/JunoOnboardingFragment.kt

```diff
@@ -169,12 +206,20 @@ class JunoOnboardingFragment : Fragment() {
         )
     }
 
-    private fun shouldShowNotificationPage(context: Context) =
+    private fun canShowNotificationPage(context: Context) =
         !NotificationManagerCompat.from(context.applicationContext)
             .areNotificationsEnabledSafe() && Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU
 
+    private fun canShowAddWidgetCard() = Build.VERSION.SDK_INT >= Build.VERSION_CODES.O
+
     private fun isNotATablet() = !resources.getBoolean(R.bool.tablet)
 
-    private fun pagesToDisplay(showNotificationPage: Boolean): List<OnboardingPageUiData> =
-        FxNimbus.features.junoOnboarding.value().cards.values.toPageUiData(showNotificationPage)
+    private fun pagesToDisplay(
+        canShowNotificationPage: Boolean,
+        canShowAddWidgetPage: Boolean,
```

Comment:
```suggestion
        showNotificationPage: Boolean,
        showAddWidgetPage: Boolean,
```
I know this suggested change was the original code. I don't think this is something we need to fix, but maybe consider this as a later follow up. I believe `showXX` was sufficient in conveying this is a boolean and aligns with how we have been naming things in our project.